packages/testing-toolkit/src/index.ts

```diff
@@ -0,0 +1,244 @@
+}
+
+export async function checkRadioOption(canvasElement: HTMLElement, linkId: string, text: string) {
+  const questionElement = await findByLinkId(canvasElement, linkId);
+  const radio = questionElement?.querySelector(`span[data-test="${text}"] input`);
+
+  if (!radio) {
+    throw new Error(`Input or textarea was not found inside ${`[data-linkid=${linkId}] block`}`);
+  }
+
+  fireEvent.click(radio);
+  // Here we await for debounced store update
+  await new Promise((resolve) => setTimeout(resolve, 500));
+}
+
+export async function getInputText(canvasElement: HTMLElement, linkId: string) {
+  const questionElement = await findByLinkId(canvasElement, linkId);
+  const input =
+    questionElement?.querySelector('input') ?? questionElement?.querySelector('textarea');
+
+  if (!input) {
+    throw new Error(`Input or textarea was not found inside ${`[data-linkid=${linkId}] block`}`);
+  }
+
+  return input.value;
+}
+
+export async function chooseSelectOption(
+  canvasElement: HTMLElement,
+  linkId: string,
+  optionLabel: string
+) {
+  const questionElement = await findByLinkId(canvasElement, linkId);
+
+  const input = questionElement.querySelector('input, textarea');
+  if (!input) {
+    throw new Error(`There is no input inside ${linkId}`);
+  }
+
+  fireEvent.focus(input);
+  fireEvent.keyDown(input, { key: 'ArrowDown', code: 'ArrowDown' });
+
+  const option = await screen.findByText(optionLabel);
+  fireEvent.click(option);
+}
+export async function chooseQuantityOption(
+  canvasElement: HTMLElement,
+  linkId: string,
+  quantity: number | string,
+  quantityComparator?: string
+) {
+  const questionElement = await findByLinkId(canvasElement, linkId);
+
+  const inputComaparator = questionElement.querySelector('div[data-test="comparator"] input');
+  const inputWeight = questionElement.querySelector('div[data-test="q-item-quantity-field"] input');
+
+  if (!inputComaparator) {
+    throw new Error(`There is no input inside ${linkId}`);
+  }
+  if (!inputWeight) {
+    throw new Error(`There is no input inside ${linkId}`);
+  }
+
+  fireEvent.focus(inputComaparator);
+  fireEvent.keyDown(inputComaparator, { key: 'ArrowDown', code: 'ArrowDown' });
+
+  if (quantityComparator) {
+    const option = await screen.findByText(quantityComparator);
+    fireEvent.click(option);
+    fireEvent.change(inputComaparator, { target: { value: quantityComparator } });
+  }
+
+  fireEvent.change(inputWeight, { target: { value: quantity } });
+  // Here we await for debounced store update
+  await new Promise((resolve) => setTimeout(resolve, 500));
+}
+
+export async function findByLinkId(canvasElement: HTMLElement, linkId: string) {
+  const selector = `[data-linkid="${linkId}"]`;
+  return await waitFor(() => {
+    const el = canvasElement.querySelector(selector);
+    if (!el) {
+      throw new Error(`Element ${selector} not found`);
+    }
+    return el;
+  });
+}
+export async function choiceOpenChoiceOther(
```

Comment:
this helper does not choose actually, inputs instead, rename to inputOpenChoiceOtherText